daemon/cluster/cluster.go

```diff
@@ -280,27 +280,30 @@ func (c *Cluster) GetDataPathAddress() string {
 	return ""
 }
 
-// GetRemoteAddress returns a known advertise address of a remote manager if
+// GetRemoteAddressList returns the known advertise address of a remote managers if
 // available.
 // todo: change to array/connect with info
-func (c *Cluster) GetRemoteAddress() string {
+func (c *Cluster) GetRemoteAddressList() []string {
```

Comment:
I would have gone with `GetRemoteAddresses`, but I guess that would take a chance to libnetwork at this point.